packages/block-editor/src/components/url-input/index.js

```diff
@@ -369,12 +370,12 @@ export default compose(
 	withSelect( ( select, props ) => {
 		// If a link suggestions handler is already provided then
 		// bail
-		if ( isFunction( props.fetchLinkSuggestions ) ) {
+		if ( isFunction( props.__experimentalFetchLinkSuggestions ) ) {
 			return;
 		}
 		const { getSettings } = select( 'core/block-editor' );
 		return {
-			fetchLinkSuggestions: getSettings().__experimentalFetchLinkSuggestions,
+			__experimentalFetchLinkSuggestions: getSettings().__experimentalFetchLinkSuggestions,
```

Comment:
This prop `__experimentalFetchLinkSuggestions` was already here before I came along:

https://github.com/WordPress/gutenberg/blob/04e142e9cbd06a45c4ea297ec573d389955c13be/packages/block-editor/src/components/url-input/index.js#L317

I'm not sure we should mark as "experimental"?

Reply:
Looking at the code in master, the un-enhanced version of `URLInput` isn't exported, so there was no way to use `fetchLinkSuggestions` previously. The value was always overridden by the wrapping `withSelect`. 

Now that the `withSelect` has an early return, the prop can be used, so I think it should be made experimental until there's more chance to think about if it's needed.

Reply:
Ah of course. I see what you mean now. Much appreciated.